tests/test_model_utilities.py

```diff
@@ -0,0 +1,27 @@
+from app.routes.model_utilities import validate_model
+from app.models.book import Book
+from werkzeug.exceptions import HTTPException
+import pytest
+
+def test_validate_model(two_saved_books):
+    # Act
+    result_book = validate_model(Book, 1)
+
+    # Assert
+    assert result_book.id == 1
+    assert result_book.title == "Ocean Book"
+    assert result_book.description == "watr 4evr"
+
+def test_validate_model_missing_record(two_saved_books):
```

Comment:
Would we want to inspect any details of the excpetion in these cases to confirm that the invalid/not found errors are handled differently?

Reply:
Good catch, we really should to differentiate better between these tests, I think I just hadn't peeled apart the object to figure out how to access useful values. Will add some asserts =]